uuid/uuid_test.go

```diff
@@ -149,3 +144,48 @@ func TestReplace(t *testing.T) {
 	}
 	tupleValueIsId(t, respSel.Data, id)
 }
+
+// runTestMain is a body of TestMain function
+// (see https://pkg.go.dev/testing#hdr-Main).
+// Using defer + os.Exit is not works so TestMain body
+// is a separate function, see
+// https://stackoverflow.com/questions/27629380/how-to-exit-a-go-program-honoring-deferred-calls
+func runTestMain(m *testing.M) int {
+	var inst test_helpers.TarantoolInstance
```

Comment:
It looks like you can replace it with `:=` for `inst`.

Reply:
Replaced